docs/src/main/asciidoc/security-using-jwt-tutorial.adoc

```diff
@@ -3,6 +3,7 @@ This guide is maintained in the main Quarkus repository
 and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
+[id="security-using-jwt-tutorial"]
```

Comment:
@MichalMaler Can we deal with renaming this file as part of the future JWT doc refactoring ?

Reply:
It just introduces a new variable, I'd rather us taking care of the whole smallrye-jwt doc refactoring in a dedicated step

Reply:
Hello @sberyozkin Thank you for your feedback. 
This is not a renaming, I just add the ID that should be there since the file creation. Heading is the same. No work needs to be done, all works the same.  This file now just have an appropriate thing that should be there when a new file is created.

Reply:
@MichalMaler Hi, I see the file name itself has changed, but it is not tutorial right now, it has everything in it, tutorial, reference. The file should remain as `security-jwt.adoc` for now IMHO

Reply:
Hello @sberyozkin . I renamed it back, also provided little tweaks when I was in the file again. The rest will need to be done with complete re-factorization of the content. Is this OK in your books now?